KMB/make_KMB_info.py

```diff
@@ -605,13 +610,46 @@ def make_default_fmis_category(self, cache):
             self.content_cats.add(
                 'Archaeological monuments in %s County' % self.lan)
 
-    def make_tag_categories(self, cache):
+    def make_item_class_categories(self, cache):
         """
-        Construct categories from the provided tags.
+        Construct categories from the item class values.
+
+        :param cache: cache for category existence
+        """
+        primary_classes = self.kmb_info.mappings['primary_classes']
+
+        # find the class/tag that is also in primary_classes
+        primary_tag = None
+        intersection = list(set(primary_classes) & set(self.item_classes))
+        if len(intersection) == 1:
+            primary_tag = intersection[0]
+        elif len(intersection) > 1:
+            pywikibot.warning(
+                "Found two primary classes. Need to rethink the logic. "
```

Comment:
This is primarily here as a safety valve since I don't yet have the real data. I don't expect it to be triggered though.